go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/prometheus/client_golang v1.18.0
 	github.com/prometheus/common v0.47.0
 	github.com/spf13/cobra v1.8.0
+	github.com/spf13/pflag v1.0.5
```

Comment:
not really sure why this moved from indirect to direct, should I revert it?

Reply:
You're now importing this dependency directly as part of your changes.

Reply:
Ah, I thought we already had some usages of pflag as we were already handling flags, but as I look we never did as it was through the cobra command.